src/raqm.c

```diff
@@ -243,6 +249,10 @@ raqm_create (void)
 
   rq->flags = RAQM_FLAG_NONE;
 
+  rq->line_width = 2147483647;
```

Comment:
A negative number would have been better, since widths can’t be negative and you are using signed integer for the width anyway.